gnome-initial-setup/pages/password/gis-password-page.c

```diff
@@ -477,7 +475,8 @@ gis_password_page_constructed (GObject *object)
                                                               "org.freedesktop.DBus.Properties",
                                                               NULL, NULL);
 
-  gtk_widget_show (GTK_WIDGET (page));
+  /* This callback shows the page widget conditionally on if it's needed */
```

Comment:
Nitpick: ‘only’?

Reply:
I think "conditional on" is a legitimate phrase